iFixit/checkout-ui-extensions
# Only show codice fiscale to Italian customers

Closes https://github.com/iFixit/checkout-ui-extensions/issues/9

Updates `@shopify/ui-extensions-react` to the latest version. A breaking change was introduced in version [`2023.10.0`](https://github.com/Shopify/ui-extensions/releases/tag/%40shopify%2Fui-extensions-react%402023.10.0) that requires us to use react 18 and add `react-reconciler` as a dependency.

## QA

- Run local dev with `pnpm dev`
- Preview `purchase.checkout.block.render`
- Make sure that the codice field only appears if your billing address company is Italy
